ethTurin track and basics (#57)
deora-earth/solarpunch: docs/teal-basics.md

> ###### tags: `tealgarden` `ethturin`

Author: remove

> # Basic Explainers Teal Garden Track :books:

Author: ```suggestion
# Teal Garden Track - Basic Explainers :books:
```

**Hackers**, welcome to our quick explainers, to give you a conception of what you need to consider building with us at **ETH Turin**.

Don't feel overwhelmed by all the materials, we simply want to give you a little primer to get started more easily :)

## What is holacracy?

**Holacracy** is a new way of structuring and running your organization that replaces the conventional management hierarchy. Instead of operating top-down, power is distributed throughout the organization – giving individuals and teams freedom while staying aligned to the organization’s purpose.

> **Holacracy-powered organizations** focus on purpose at every level of scale: organizational purpose, team purpose, and individual purpose are all explicit and aligned. Every team member directs their energy in alignment with the broader mission, unlocking your organization’s full potential.

Author: maybe add a link to further information. either the holacracy constitution, or a list of organizations that practice holacracy, so that the reader can give the concept some credability and aknowledge it as proven concept, rather than thinking that it is some pie in the sky idea.

## What is a Teal Organization?

Characterized by **self-management, wholeness, and a deeper sense of purpose**, these organizations – which are described as **“Teal”** – operate largely without organization charts, management hierarchies, quarterly goals or other traditional management strategies.

> Instead, they’re characterized by features like **self-managed teams**, intuitive reasoning and **decentralized decision-making**. Because these practices are so new, the conversation around what they are and how to implement them is **ongoing**.

Author: also add a link here. maybe to the teal wiki, or a showcase from the book.

> ## What is a process ?

Author: maybe move this one up first, because this is what we want from them?

> ## What is a process ?

Author: ```suggestion
## What is a process?
```

A simple Wikipedia entry can help us understand what a process is:

> “A business process or business method is a collection of related, structured activities or tasks by people or equipment in which a specific sequence produces a service or product (serves a particular business goal) for a particular customer or customers.”

> We created an evaluation page to make sure we & you know what a good process is all about:

Author: maybe list quickly how a process is structured in teal garden. (categories, components...)

> - [What is a valid submission to teal.garden?](https://hackmd.io/dMlEYrGOTIm8Yw4e50V8qA?view)

Author: ```suggestion
- [What is a valid submission to teal.garden?](https://github.com/deora-earth/tealgarden/tree/develop/docs/submission-evaluation.md)
```

> ## What is a component ?

Author: ```suggestion
## What is a component?
```

The components are the building blocks of our process. Most of the time, a simple list is sufficient, to make sure we have all requirements in place.

A component can be **a role, a policy, or a tool**. They can be exisiting in the organizations setup, or will be added during the process. All these preparing investigations help understand the dynamics of the components and applying them in a new setup.

**Roles & Policies are connected to People.
Tools are connected to Technology.**

> Anyone who wants to adopt your process will need to follow the instructions to setup a tool. It consist of one or several steps. Each step decribes how to integrate one tool into the process. Make sure to document each component which is related to a tool.

Author: maybe move all this to instructions. just say: if you have a component that is a tool, you will have to give instructions how to use that tool in the `instructions` section.

> ## What is a adoption requirement ?

Author: ```suggestion
## What is a adoption requirement?
```

While collecting ideas and checking the components, ask yourself the following questions:

- What is essential to perform this process?
- Which prerequisites are needed to adopt this process as practice?

**Examples:**

- "The organization needs to practice holacracy v4.1 or higher"

or:

- "Each team-member needs a web-3 enabled wallet."

> ## What is a Role ?

Author: ```suggestion
## What is a Role?
```

### without context of holacracy:

A "role" as the set of expectations that society places on an individual. By unspoken consensus, certain behaviors are deemed "appropriate" and others "inappropriate".

**Example:**

> "An "appropriate" doctor dresses fairly conservatively, asks a series of personal questions about one's health, touches one in ways that would normally be forbidden, writes prescriptions, and shows more concern for the personal wellbeing of his or her clients than is expected of, say, an electrician or a shopkeeper."

Roles are not limited to occupational status, of course, nor does the fact that one is cast in the role of "doctor" during working hours prevent one from taking on other roles at other times: husband/wife, friend, father/mother, and so on.

> **-- Roles are the expectations of society placed on the individual.**

Author: needed?

> ### in context of holacracy:

Author: ```suggestion
### in context of teal organizations:
```

Author: maybe we put the docs more in the light of teal organizations? because holacracy is kind of special.

The building blocks of Holacracy's organizational structure are roles. Holacracy distinguishes between roles and the people who fill them, as one individual can hold multiple roles at any given time.

A role is not a job description; its definition follows a clear format including a name, a purpose, optional "domains" to control, and accountabilities, which are ongoing activities to perform.

Roles are defined by each circle —or team— via a collective governance process, and are updated regularly in order to adapt to the ever-evolving needs of the organization.

-- **Roles are defined and chosen by the individual itself.**

> ## What is a Policy ?

Author: ```suggestion
## What is a Policy?
```

A **Policy** is created by a Circle or Role to grant or limit access over some Domain that it controls. Using Policies, self-managed teams are also able to setup their own system of rewarding & penalizing certain behaviours.

**Here is a example policy:**

> Policy: Password Requirements
> "Anyone using passwords to secure company-related accounts with access to company assets or sensitive data must ensure those passwords are strong, not duplicated on any other system, not stored anywhere except a highly protected repository, and may not share the actual password text with anyone else."

Policies are proposed by the roles which feel a need to implement them. The governance process of holacracy can help enabling these policies for the organization.

-- **Draft a simple policy if it is necessary for your process, make sure to mention it in the instructions.**

> ## What is a Domain ?

Author: ```suggestion
## What is a Domain?
```

A Domain is a "property" of the organization (e.g. asset, function, or process) given to a role or circle in order to give it exclusive control of that resource.

**Examples of domains:**

- The company's main website
- Event registration process
- Financial transaction processing systems & processes

## :tada: Bonus:

> **Whats up with these colors ?**

Author: ```suggestion
**Whats up with these colors?**
```

**Frederic Laloux’s** acclaimed book _"Reinventing Organisations"_ describes how every time humanity shifts to a new stage of evolution it also achieves a radically more productive organisational model.

**Organisational Categories**
Laloux’s research consolidates and builds upon previous work, setting it all within a framework that is easier to understand. Each paradigm shift in organisation is identified and categorised with the ones we see today being shown in the following table:

![Colors of organizations](https://i.imgur.com/rbkvzdp.png)

**Want to learn [more](http://pragmaticscrum.info/what-colour-is-your-organisation/)?**

---

> \***\*--> End of explainers <--\*\***

Author: add a link to discord, and tell them to reach out to support.
